### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,64 @@
...
-git submodule update --init --recursive
-git push -u origin master
-```
-4. `mkdir build && cd build`
-5. `cmake .. && make`
-6. `ctest`
+CSM-CameraModel contains two different sensor models. The first, is a generic
+framing camera model written from scratch. The second is a generic line scan
+camera model based on code from BAE Systems Information and Electronic Systems
+Integration Inc.
```
Comment (thareUSGS) [inline README.md]: Inc. (see LICENSE file).

Comment (thareUSGS) [reply README.md]: Not sure how to handle our public domain and BAE's BSD_like license. Should we put the BAE license next to the BAE code? Here is the BSD license we need to put somewhere: https://www.dropbox.com/s/nfilxbcqkdmh5hr/license.txt?dl=0

Comment (jessemapel) [reply README.md]: Are you sure? I copied their name straight from the comment block they put at the top of the source files.

File: README.md [hunk 2]
```diff
@@ -1,100 +1,64 @@
...
+From the CSM Plugin interface, a generic framing camera model
+(USGS_ASTRO_FRAME_SENSOR_MODEL) or generic line scan camera model
+(USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL) can be instantiated from suitable Image
+Support Data (ISD). Under the CSM standard, each plugin library can define its
+own ISD format. This library uses an auxiliary JSON formatted file that must be
+next to the image file passed to the CSM::ISD class. We provide an OpenAPI
+server for generating these,
+[pfeffernusse](https://github.com/USGS-Astrogeology/pfeffernusse). The swagger
+specification is located on
+[swaggerhub](https://app.swaggerhub.com/apis/USGS-Astro/pfeffernusse2/0.1.4).
```
Comment (kberryUSGS) [inline README.md]: Should this be: https://app.swaggerhub.com/apis/USGS-Astro/pfeffernusse2/0.1.4-oas3 ?

**Comment by jessemapel** [comment]
@thareUSGS @kberryUSGS I think this is ready to go now

**Pull request merged**